Primo.js

```diff
@@ -25,6 +25,7 @@ Primos with showPNX.jsp installed:
 (3) http://limo.libis.be/primo_library/libweb/action/search.do?vid=LIBISnet&fromLogin=true
 (4.a) http://virtuose.uqam.ca/primo_library/libweb/action/search.do?vid=UQAM
 (5) http://searchit.princeton.edu/primo_library/libweb/action/dlDisplay.do?docId=PRN_VOYAGER2778598&vid=PRINCETON&institution=PRN
+(6) http://digitale.beic.it/primo_library/libweb/action/search.do?vid=beic
```

Comment:
That is not the same, see http://www.exlibrisgroup.org/display/PrimoCC/showPNX+Revisited